cmd/pack/main.go

```diff
@@ -45,14 +45,19 @@ func main() {
 				if flag, err := fs.GetBool("timestamps"); err == nil {
 					logger.WantTime(flag)
 				}
+				if flag, err := fs.GetBool("experimental"); err == nil {
+					experimental = flag
+				}
 			}
 
 			packClient = initClient(logger)
+			packClient.Experimental(experimental)
 		},
 	}
 
 	rootCmd.PersistentFlags().Bool("no-color", false, "Disable color output")
 	rootCmd.PersistentFlags().Bool("timestamps", false, "Enable timestamps in output")
+	rootCmd.PersistentFlags().Bool("experimental", false, "Enable experimental features")
```

Comment:
I'm curious what the expected experience for this type of implementation. The initial proposal might not be clear enough but the initial intent would be that it's a persisted state similar to the [docker CLI](https://docs.docker.com/engine/reference/commandline/cli/#experimental-features).